packages/data-context/src/actions/ProjectActions.ts

```diff
@@ -226,7 +232,9 @@ export class ProjectActions {
       return null
     }
 
-    const spec: Cypress.Spec = {
+    // launchProject expects a spec when opening browser for url navigation.
+    // We give it an empty spec if none is passed so as to land on home page
+    const emptySpec: Cypress.Spec = {
```

Comment:
I wonder if we still need this? You could try removing it and see what happens - I don't know if this is necessary now we launch into unified (no need for a fake spec url to visit).

Reply:
We still need this. It could be removed but it would require refactoring a chunk of server code that is outside the scope of this PR.